CONTRIBUTING.md

```
@@ -5,71 +5,77 @@
+```
+
+To check the `pixi` lock file is up to date:
+
+```console
+pixi run check-lock
 ```
 
 ## Shortcut Task Commands
 
-###### For Running Individual Checks
+###### For Running Individual Toolkit Checks
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)